docs/source/models/causes/neonatal/preterm_birth.rst

```diff
@@ -154,17 +154,31 @@ Data Tables
 The Preterm Birth model requires only the probability of death (aka "mortality risk") for use
 in the decision graph. This will be computed from the overall neonatal mortality risk and the cause-specific mortality fraction.
 
-Since this is a PAF-of-one cause, the calculation must take into account the "structural zeros" representing no mortality risk for simulants withe a gestational age above 37 weeks.
+Since this is a PAF-of-one cause, the calculation must take into account the "structural zeros" representing no mortality risk for simulants with a gestational age of 38 or more weeks.
 
-The details of this calculation require information about other subcauses as well as preterm, and therefore are included in the :ref:`Overall Neonatal Disorders Model <2021_cause_neonatal_disorders_mncnh>` page.
+The details of this calculation require information about other subcauses as well as preterm, and therefore are included in the :ref:`Overall Neonatal Disorders Model <2021_cause_neonatal_disorders_mncnh>` page.  This page describes the birth-weight- and gestational-age-specific cause specific mortality rates that are used for this cause on that page, :math:`\text{CSMR}^{\text{preterm with RDS}}_{\text{BW},\text{GA}}` and :math:`\text{CSMR}^{\text{preterm without RDS}}_{\text{BW},\text{GA}}`. In both cases, the formula is:
+
+.. math::
+    \begin{align*}
+    \text{CSMR}^{k}_{\text{BW},\text{GA}}
+    &=
+    \begin{cases}
+    \text{CSMR}\cdot f_k \cdot \text{RR}_{\text{BW},\text{GA}} \cdot Z, & \text{if BW} < 38; \\
+    0, & \text{if BW} \geq 38;
+    \end{cases}
+    \end{align*}
+
+where :math:`k` is the subcause of interest (preterm birth with or without RDS),
+:math:`\text{CSMR}` is the cause-specific mortality rate for preterm birth complications,
+:math:`f_k` is the fraction of preterm deaths due to subsubcause :math:`k` (with or without RDS), :math:`\text{RR}_{\text{BW},\text{GA}}` is the relative risk of all-cause mortality for a birth weight of :math:`\text{BW}` and gestational age of :math:`\text{GA}`, and :math:`Z` is a normalizing constant selected so that :math:`\int_{\text{BW<38}} \int_{\text{GA}} \text{RR}_{\text{BW},\text{GA}} \cdot Z = 1`.
```

Comment:
I think it's important to mention somewhere that the choice to use $\text{RR}\_{\text{BW,GA}}$ in this equation is essentially arbitrary, and it could be replaced by any other "weight function" $w(\text{BW,GA})$ as long it doesn't lead to a negative "other causes" mortality hazard.

I think it's important to mention this because, since there are a lot of pieces that go into this, it's useful to know which pieces are essential to getting it right and which ones aren't. Also, if we get more specific data about RDS death rates stratified by gestational age, we may want to change these weights to match that.

Also, I believe it is possible that the choice of $\text{RR}\_{\text{BW,GA}}$ might not work for every subcause. Since we're moving all the preterm mortality into the preterm categories, there is less room there for mortality from other causes, so depending on the hazards involved, we may need to shift mortality from some other causes into the non-preterm categories in order to avoid making things negative.

In fact, it's conceivable that there's no way to make this work at all (i.e., any choice of weight function leads to negative mortality hazards) if the GBD LBWSG data is inconsistent with the GBD preterm birth data. Hopefully that won't be an issue, but we haven't actually tried it with the real data yet.

Reply:
I will add this as a note in the docs